src/components/Socials/styles.scss

```
@@ -1,3 +1,83 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em; //nothing comment to make sure class appears
+	display: flex;
+	justify-content: space-between;
+	font-size: 2rem;
+	
+	&.closed {
+		opacity: 0;
+	}
+
+	&.open {
+		width: 50%;
+		opacity: 1;
+	}
+	
+		.item {
+			width: 1em;
+			height: 1em;
+			list-style: none;
+		}
+
+			.link {
+				display: block;
+				width: 100%;
+				height: 100%;
+				background-color: blue;
+				background-size: contain;
+				background-position: center;
+				background-repeat: no-repeat;
+
+				&.facebook {
+					background-image: url("path/to/facebook.svg");
+				}
+
+				&.twitter {
+					background-image: url("path/to/twitter.svg");
+				}
+
+				&.instagram {
+					background-image: url("path/to/instagram.svg");
+				}
+			}
 }
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.wrapper.navOpen {
```

```suggestion
	.wrapper.open {
```
aah, I think these are failing because it's `open` instead of `navOpen` now - don't forget to test after you make changes! 🙌